### PR TITLE
Assignment improvements

### PR DESCRIPTION
**Pull request by LinMAD**
Refining the repository with better instructions and simplifying project structure.

### CODE REVIEW

File: docs/instructions.md [hunk 1]
```diff
@@ -29,32 +47,46 @@ ___
...
+Implement an API sever that provides the following service.
+
+- If you know gRPC: [proto-file](../api/v1/logistics.proto)
+- Alternative HTTP: [swagger-file](../api/v1/logistics.swagger.json)
 
-- The solution should output a log message to STDOUT
-once per second with the number of received messages per second.
+> ! The solution must continuously output a log message to STDOUT every second.
+> Each log message should include the total number of messages received during
+> that second.
```
Comment (nhhagen) [inline docs/instructions.md]: Using Markdown quotes combined with how GitHub renders them does not bring the attention you intended to this information. I would use a normal paragraph.

```suggestion
The solution must continuously output a log message to STDOUT every second.
Each log message should include the total number of messages received during
that second.
```

File: README.md [hunk 2]
```diff
@@ -1,45 +1,57 @@
...
+> This repository hosts the source code for a client designed to generate a
+> specified number of requests (N).
+>
+> As part of the assignment, the candidate is
+> required to develop an API server capable of handling these requests.
+> The server can be implemented using either gRPC or HTTP protocols.
+> 
+> The primary objective of this test is to assess the candidate's ability to
+> create a robust API server that efficiently processes incoming requests as
+> specified by the client's operations
```
Comment (nhhagen) [inline README.md]: Using Markdown quotes combined with how GitHub renders them does not bring the attention you intended to this information. I would use a normal paragraph.

```suggestion
This repository hosts the source code for a client designed to generate a
specified number of requests (N).

As part of the assignment, the candidate is
required to develop an API server capable of handling these requests.
The server can be implemented using either gRPC or HTTP protocols.
 
The primary objective of this test is to assess the candidate's ability to
create a robust API server that efficiently processes incoming requests as
specified by the client's operations
```

File: docs/instructions.md (hunk 1 above)
Comment (LinMAD) [reply docs/instructions.md]: Made it even better then - **Bold**

File: README.md (hunk 2 above)
Comment (LinMAD) [reply README.md]: Made it even better then - **Bold**

**Comment by guillermoBallester** [comment]
@LinMAD I think the instructions are now clearer and more straightforward for the participants to understand what they need to do 👍

**Comment by ezorrio** [comment]
@LinMAD
Definitely more understandable, I'd say in some places there are more details than it's supposed to be, but always nice to have those in place:)

LGTM

File: README.md [hunk 3]
```diff
@@ -1,45 +1,57 @@
...
-├── api                                  // API definition with gRPC proto-files
-├── docker-compose                       // Docker related files
-├── docs                                 // Instructions
-└── src                                  // Source code related to test assignment
-    ├── client                           // Client that will send requests to API Server
-    ├── generated                        // Shared code that can be used to implement gRPC
-    │   └── logistics
-    │       └── api
-    │           └── v1                   // protoc-gen-go of API
-    └── server                           // Your solution that provides API server
```
Comment (geethu-mohan) [inline README.md:31]: minor: maybe also include this. `server   // Your solution that provides API server`

**Review by geethu-mohan — APPROVED** [review]

**Pull request merged**